exporter/opentelemetry-exporter-otlp-json-common/src/opentelemetry/exporter/otlp/json/common/_internal/_log_encoder/__init__.py

```diff
@@ -0,0 +1,271 @@
+
+    if (
+        hasattr(log_record, "trace_flags")
+        and log_record.trace_flags is not None
+    ):
+        result["flags"] = int(log_record.trace_flags)
+
+    return result
+
+
+def _encode_attributes(attributes: Dict[str, Any]) -> List[Dict[str, Any]]:
+    """Encodes attributes into OTLP JSON format."""
+    if not attributes:
+        return []
+
+    attribute_list = []
+    for key, value in attributes.items():
+        if value is None:
+            continue
+
+        attribute = {"key": key}
+        attribute.update(_encode_attribute_value(value))
+        attribute_list.append(attribute)
+
+    return attribute_list
+
+
+# pylint: disable=too-many-return-statements
+def _encode_attribute_value(value: Any) -> Dict[str, Any]:
+    """Encodes a single attribute value into OTLP JSON format."""
+    if isinstance(value, bool):
+        return {"value": {"boolValue": value}}
+    if isinstance(value, int):
+        return {"value": {"intValue": value}}
+    if isinstance(value, float):
+        return {"value": {"doubleValue": value}}
+    if isinstance(value, str):
+        return {"value": {"stringValue": value}}
+    if isinstance(value, (list, tuple)):
+        if not value:
+            return {"value": {"arrayValue": {"values": []}}}
+
+        array_value = {"values": []}
+        for element in value:
+            element_value = _encode_attribute_value(element)["value"]
+            array_value["values"].append(element_value)
+
+        return {"value": {"arrayValue": array_value}}
+    if isinstance(value, bytes):
+        return {
+            "value": {"bytesValue": base64.b64encode(value).decode("ascii")}
+        }
+    # Convert anything else to string
+    return {"value": {"stringValue": str(value)}}
+
+
+# pylint: disable=too-many-return-statements
+def _encode_any_value(value: Any) -> Dict[str, Any]:
+    """Encodes any log record body value into OTLP JSON format."""
+    if isinstance(value, bool):
+        return {"boolValue": value}
+    if isinstance(value, int):
+        return {"intValue": str(value)}
+    if isinstance(value, float):
+        return {"doubleValue": value}
+    if isinstance(value, str):
+        return {"stringValue": value}
+    if isinstance(value, (list, tuple)):
+        values = []
+        for element in value:
+            values.append(_encode_any_value(element))
+        return {"arrayValue": {"values": values}}
+    if isinstance(value, dict):
+        kvlist = []
+        for key, val in value.items():
+            if val is not None:
+                kv = {"key": str(key)}
+                kv.update(_encode_any_value(val))
+                kvlist.append(kv)
+        return {"kvlistValue": {"values": kvlist}}
+    if isinstance(value, bytes):
+        return {"bytesValue": base64.b64encode(value).decode("ascii")}
+    # Convert anything else to string
+    return {"stringValue": str(value)}
+
+
+def _get_severity_number_value(severity_number: SeverityNumber) -> str:
```

Comment:
Please refer to https://opentelemetry.io/docs/specs/otlp/#json-protobuf-encoding

The enum values are encoded as ints, not labels.